runtime/lua/vim/_meta/api.lua

```diff
@@ -744,11 +744,6 @@ function vim.api.nvim_buf_set_option(buffer, name, value) end
 --- `start_row = end_row = row` and `start_col = end_col = col`. To delete the
 --- text in a range, use `replacement = {}`.
 ---
---- Prefer `nvim_buf_set_lines()` if you are only adding or deleting entire
---- lines.
----
---- Prefer `nvim_put()` if you want to insert text at the cursor position.
----
```

Comment:
Looks like our meta docstrings don't have `@note`, I assume that isn't intentional @lewis6991 ?

Reply:
You mean we don't use `@note` or that it isn't implemented?

Reply:
`@note` items are added to generated vimdocs, but not _meta doctsrings.

Reply:
ah, that's not intentional.